src/zeit/content/cp/testing.py

```diff
@@ -64,8 +66,29 @@ def testTearDown(self):
 
 SOLR_MOCK_LAYER = SolrMockLayer()
 
+
+# We cannot use a layer from zeit.retresco.testing because importing that
```

Comment:
OK. Ich glaub, jetzt ist der Punkt gekommen, wo wir uns für die Kreisabhängigkeiten von/zu solr bzw retresco was schlaueres einfallen lassen sollten. In zeit.push missbrauchen wir die `__init__.py` um Testkram abzulegen. Find ich nicht optimal, aber falls uns sonst nix einfällt, wär mir das denk ich trotzdem noch lieber.